astropy/io/fits/convenience.py

```diff
@@ -990,8 +992,8 @@ def _getext(filename, mode, *args, ext=None, extname=None, extver=None,
     """
 
     err_msg = ('Redundant/conflicting extension arguments(s): {}'.format(
```

Comment:
you broke with `.format()` above, and that seemed more logical, maybe do it here, too

Reply:
All these changes are only indentation fixes by autopep8, there are too many lines where `.format` could be break as you suggest (which I usually prefer).